### PR TITLE
Add url dispatcher benchmark for resolving root route for github simulated routes tree

### PR DESCRIPTION
**Pull request by asvetlov**
More benchmarks is better!

### CODE REVIEW

**Review by bdraco — APPROVED** [review]

**Comment by bdraco** [comment]
there is a mypy nit
```
tests/test_benchmarks_web_urldispatcher.py:33:5: error: Returning Any from
function declared to return "list[str]"  [no-any-return]
        return urls
```

**Comment by asvetlov** [comment]
Yep, I see. New commit should fix it.

**Comment by bdraco** [comment]
Need to merge https://github.com/aio-libs/aiohttp/pull/10020 first.  Will take care of merging this one as soon as 10020 finishes

File: tests/test_benchmarks_web_urldispatcher.py [hunk 1]
```diff
@@ -18,6 +19,20 @@
...
+    """GitHub api urls."""
+    # The fixture provides OpenAPI generated info for github.
+    # To update the local data file please run the following command:
+    # $ curl https://raw.githubusercontent.com/github/rest-api-description/refs/heads/main/descriptions/api.github.com/api.github.com.json | jq ".paths | keys" > github-urls.json
+
+    here = Path(__file__).parent
+    with (here / "github-urls.json").open() as f:
+        urls = json.load(f)
+
+    return cast(list[str], urls)
```
Comment (Dreamsorcerer) [inline tests/test_benchmarks_web_urldispatcher.py:33]: We shouldn't cast, it bypasses any type safety that may be present and could produce problems later on. Here we can just do:
```
        urls: list[str] = json.load(f)
```

**Pull request merged**